cmd/systray/systray.go

```diff
@@ -110,24 +125,47 @@ func main() {
 	defer runtime.UnlockOSThread()
 
 	flag.BoolVar(&launchgui, "launch-gui", false, "Launch browser configuration and exit")
+
+	// launch-elev=true only means the process should have been elevated so that it will not elevate again. If the
+	// parameter is specified but the process is not elevated, some operation will fail due to access denied.
+	flag.BoolVar(&launchelev, "launch-elev", false, "Launch program as elevated, internal use only")
```

Comment:
Because end user should not use this parameter. It's used by the elevation logic to prevent it from elevation dead loop in case something goes wrong.

Reply:
Btw, is there a way to show command line usage in ddtray? I tried -help but it didn't show anything and just started the app. Since it's a internal-ish flag, I tend to keep launch-elev.